tests/db_utils/tests.py

```diff
@@ -83,3 +84,29 @@ def test_load_backend_invalid_name(self):
         with self.assertRaisesMessage(ImproperlyConfigured, msg) as cm:
             load_backend("foo")
         self.assertEqual(str(cm.exception.__cause__), "No module named 'foo'")
+
+
+def connection_is_open():
+    return connection.is_usable()
+
+
+class DatabaseConnectionForkedProcessTests(SimpleTestCase):
+    databases = ["default"]
+
+    @unittest.skipUnless(connection.vendor == "sqlite3", "SQLite are always usable")
+    def test_connections_closed_on_forked_process(self):
+        connection = connections[DEFAULT_DB_ALIAS]
+
+        # check connection
+        with connection.cursor() as cursor:
+            cursor.execute("SELECT 1")
+
+        # create multiprocessing pool
+        pool = multiprocessing.Pool(1)
+
+        is_open = pool.apply(connection_is_open)
+        pool.close()
+        pool.join()
+
+        # check that the connection is closed
+        self.assertEqual(is_open, False)
```

Comment:
We should also assert that the connection in the parent/current process remains usable.

```suggestion
        self.assertIs(connection.is_usable(), True)
```